static/stable/stage/navigation/iam-navigation.json

```diff
@@ -202,8 +202,10 @@
       ]
     },
     {
+      "id": "authentication-policy",
       "title": "Authentication Policy",
       "expandable": true,
+      "feoReplcement": "authentication-policy",
```

Comment:
@zwitter1 FWIW you'll probably need to add this to the prod config in the future as well. You can add it before hand and nothing should happen until you deploy IAM to prod with the `feoConfigEnabled`/updated frontend CRD. Just an FYI